static/css/style.css

```diff
@@ -48,6 +48,16 @@ body {
     padding-top:18px;
 }
 
+#response {
+    background-color: #e6eaff;
+    border: 1px solid #716f6f;
+    padding-right: 20px;
+    padding-left: 5px;
+}
+
+#response p {
+    font-style: italic;
+}
```

Comment:
this is what it looks like before winning (the line below the guess box) - can we remove that?

<img width="908" alt="image" src="https://user-images.githubusercontent.com/11841271/159566668-e130c4be-6fb2-40a2-9357-ee23c0fd70d3.png">

Reply:
Right, that's something that is probably not possible to change with only CSS and I unfortunately don't do Python :) Can drop the border.